lib/tasks/publishing_api.rake

```diff
@@ -5,8 +5,10 @@ require "gds_api/publishing_api/special_route_publisher"
 namespace :publishing_api do
   desc "Publish special routes via publishing api"
   task publish_special_routes: :environment do
+    content_id = "bce40c1f-2259-4404-b275-8c5e04afef34".freeze
```

Comment:
This freeze seems unusual - I don't think we gain from it. Same in the test file.

Reply:
Yeah, I originally had it as a constant (CONTENT_ID). The linter made me do the `.freeze`, and then I changed my mind about the constant. I'll tidy this up.